idd/Energy+.idd.in

```diff
@@ -30616,9 +30616,33 @@ Sizing:Zone,
       \minimum 0
       \default 0.3
       \note This input is used for autosizing components when Heating Design Air Flow Method = DesignDayWithLimit.
-  A7; \field Design Specification Zone Air Distribution Object Name
+  A7, \field Design Specification Zone Air Distribution Object Name
       \type object-list
       \object-list DesignSpecificationZoneAirDistributionNames
+  A8, \field Account for Dedicated Outside Air System
```

Comment:
@wfbuhl Outside should be Outdoor everywhere for consistency with other field names.